synfig-studio/src/gui/workarea.h

```diff
@@ -206,7 +206,7 @@ class WorkArea : public Gtk::Grid, public Duckmatic
 	int dirty_trap_queued;
 
 	// This flag is set if onion skin is visible
-	bool onion_skin;
+	bool onion_skin;	‘studio::WorkArea::onion_skin’ will be initialized after [-Wreorder]
```

Comment:
The warning messages were about the object's member initialization order...
You should have at least basic knowledge of OOP to understand and resolve the issue

Reply:
Kindly tell me what is and how to use wreorder 

Thanks :)